TheMovieDb/Modules/MovieDetail/MovieDetailRemoteDataManager.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MovieDetailRemoteDataManager.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 14/11/21.
+//  
+//
+
+import Foundation
+
+class MovieDetailRemoteDataManager: MovieDetailRemoteDataManagerInputProtocol {
```

Comment:
take a look at facade, it's pretty much what you are doing here :) good work
https://refactoring.guru/design-patterns/facade